.github/pull_request_template.md

```diff
@@ -0,0 +1,31 @@
+<!--
+Use this template as a guide. Omit sections that don't apply.
+More details about the template are at https://github.com/edx/open-edx-proposals/pull/180
+-->
+
+## Description
+
+Describe what this pull request changes, and why. Include implications for people using this change.
+Design decisions should be documented in the repo (docstring / README / ADR) and linked here.
+
+Useful information:
+- Which edX user roles will this change impact? Common user roles are "Learner", "Course Author",
+"Developer" and "Admin".
+- How can it be tested?
+- Include screenshots for changes to the UI (ideally, both "before" and "after" screenshots, if applicable).
+- Configuration changes needed.
```

Comment:
Instead of having people include config info in the PR description, what if we remind them to annotate their configs and link to the config annotations?  Robert may have a suggestion for this.

Reply:
"Provide links to the description of corresponding configuration changes. Remember to correctly annotate these changes."